tony-core/src/main/resources/tony-default.xml

```diff
@@ -261,6 +261,18 @@
     finished directory.</description>
   </property>
 
+  <property>
+    <name>tony.history.retention-sec</name>
+    <value>2592000</value>
+    <description>How many seconds to retain history files for.</description>
```

Comment:
can you put how many days is 2592000 in the desc? same below

Reply:
Good idea. Updated.